### PR TITLE
Add select to bottom

### PR DESCRIPTION
**Pull request by Albacusphetical**
Simply allows the user to select from a note to the bottom of the sheet, which can be useful for multi-transposition from certain points

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -566,6 +566,12 @@
         renderSelection()
     }
 
+    function selectToBottom() {
+        selection.right = chords_and_otherwise.length - 1
+
+        renderSelection()
+    }
+
```
Comment (ArijanJ) [inline src/App.svelte]: You can remove this function, can be done inline
```suggestion
```

File: src/App.svelte [hunk 2]
```diff
@@ -592,6 +598,11 @@
 
     function setSelection(event_or_index) {
         let index = event_or_index.detail?.index ?? event_or_index
+        if (event_or_index.detail?.toBottom) {
```
Comment (ArijanJ) [inline src/App.svelte:595]: There's a bug where the previous regular selection stays selected if you select a new one with this method:
```suggestion
        if (event_or_index.detail?.toBottom) {
                resetSelection()
```

File: src/App.svelte [hunk 3]
```diff
@@ -592,6 +598,11 @@
...
+            selection.left = index
+            selectToBottom()
+            return;
```
Comment (ArijanJ) [inline src/App.svelte]: The function can continue here with hijacked parameters, `renderSelection` also gets called at the end of it anyway
```suggestion
            index = chords_and_otherwise.length - 1
```

**Review by ArijanJ — CHANGES_REQUESTED** [review]
Looks good, just fix the bug and see if you you want to accept the other changes

**Comment by Albacusphetical** [comment]
Nice catch 👍, I have added as suggested

**Comment by ArijanJ** [comment]
Perfect, thanks!

**Pull request merged**